src/System.Net.Mail/src/System/Net/DelegatedStream.cs

```diff
@@ -14,9 +14,6 @@ internal class DelegatedStream : Stream
         private readonly Stream _stream;
         private readonly NetworkStream _netStream;
 
-        protected DelegatedStream()
```

Comment:
@stephentoub is there any compat impact of removing a protected constructor? eg., user code that calls `base()`?